powerdns/models/powerdns.py

```diff
@@ -210,6 +210,10 @@ class Domain(PreviousStateMixin, OwnershipByService, TimeTrackable, Owned):
             "to it without owner's permission?"
         )
     )
+    owners = models.ManyToManyField(
```

Comment:
what about naming that we've discussed yesterday? To name this field other than `owners` (ex. `direct_owners`) and to name the method currently named `all_owners` `owners` (it could be property IMO)